templates/projects/classlib/project.json

```diff
@@ -10,11 +10,11 @@
     "dnx451": { },
     "dnxcore50": {
       "dependencies": {
-        "Microsoft.CSharp": "4.0.1-beta-23217",
-        "System.Collections": "4.0.11-beta-23217",
-        "System.Linq": "4.0.1-beta-23217",
-        "System.Runtime": "4.0.21-beta-23217",
-        "System.Threading": "4.0.11-beta-23217"
+        "Microsoft.CSharp": "4.0.1-beta-*",
```

Comment:
@rustd same question here, what's the final package version that should be here?